monitorix

```diff
@@ -223,51 +223,48 @@ EOF
 	if(!open(OUT, "> $config{base_dir}/index.html")) {
 		die "unable to create '${config{base_dir}}index.html': $!";
 	}
+	my $css = sprintf("%scss/monitorix-%s.css", $base_url, $config{theme_color});
```

Comment:
The names of the themes don't need to be prefixed as `monitorix-` it's redundant.

Reply:
fixed